bundles/net.tourbook.cloud/src/net/tourbook/cloud/messages.properties

```diff
@@ -1,103 +1,89 @@
+Log_DownloadWorkoutsFromSuunto_007_Error                   = The following error occurred when retrieving the workout of Id {0}: {1}
 
-Image__Connect_With_Strava = btn_strava_connectwith_light.png
+Log_UploadRoutesToSuunto_001_Start           = Uploading {0} Routes to Suunto App
+Log_UploadRoutesToSuunto_002_NoGpsCoordinate = {0} -> GPS coordinates are missing
+Log_UploadRoutesToSuunto_003_UploadStatus    = {0} -> Uploaded Route Id: "{1}"
+Log_UploadRoutesToSuunto_004_UploadError     = {0} -> Error while uploading the route: "{1}"
 
 Log_UploadToursToStrava_001_Start        = Uploading {0} tours to Strava
 Log_UploadToursToStrava_002_NoTourTitle  = {0} -> Tour title is missing
 Log_UploadToursToStrava_003_ActivityLink = {0} -> Uploaded Activity Link: <br><a href="https://www.strava.com/activities/{1}">https://www.strava.com/activities/{1}</a></br>
 Log_UploadToursToStrava_003_UploadStatus = {0} -> Upload Id: "{1}". Creation Activity Status: "{2}"
 Log_UploadToursToStrava_004_UploadError  = {0} -> Error while uploading the tour: "{1}"
 
-PrefPage_Account_Information_Label_AthleteName=Athlete's name
-PrefPage_Account_Information_Label_AthleteWebPage=Athlete's web page
-
-Dialog_UploadTours_SubTask={0} Tour Preparation\n\
-                              {1} Upload
-
-Dialog_UploadTours_Task=Uploading {0} tours to Strava ({1})
+PrefPage_AccountInformation_Label_AthleteName      = Athlete's name
+PrefPage_AccountInformation_Label_AthleteWebPage   = Athlete's web page
+PrefPage_AccountInformation_Link_Strava_WebPage    = https://www.strava.com
+PrefPage_AccountInformation_Link_SuuntoApp_WebPage = https://www.suunto.com/suunto-app/suunto-app/
+
+PrefPage_CloudConnectivity_Button_Authorize            = &Authorize
+PrefPage_CloudConnectivity_Dropbox_AccessToken_Tooltip = This is the access token to access your Dropbox account
+PrefPage_CloudConnectivity_Dropbox_WebPage_Link        = https://www.dropbox.com
+PrefPage_CloudConnectivity_Group_CloudAccount          = Cloud Account
+PrefPage_CloudConnectivity_Label_AccessToken           = Access token
+PrefPage_CloudConnectivity_Label_ExpiresAt             = Expires at
+PrefPage_CloudConnectivity_Label_RefreshToken          = Refresh token
+PrefPage_CloudConnectivity_Label_WebPage               = Web page
+PrefPage_CloudConnectivity_UnavailablePort_Message     = The port {0} necessary to retrieve the {1} token is unavailable or already in use.
+PrefPage_CloudConnectivity_UnavailablePort_Title       = Port Unavailable
+
+PrefPage_SuuntoWorkouts_Checkbox_SinceDateFilter = &Use a date filter
+PrefPage_SuuntoWorkouts_FolderPath_Tooltip       = This is the folder where the new workout files will be downloaded to
+PrefPage_SuuntoWorkouts_Label_FolderPath         = Download &folder
+PrefPage_SuuntoWorkouts_SinceDateFilter_Tooltip  = Only workouts for which the start date is greater than this date will be downloaded
+
+Suunto_WorkoutsDownloader_Tooltip = Download workouts from a Suunto App account
```

Comment:
`Suunto_WorkoutsDownloader_Tooltip`

All other text keys in the same dialog have this prefix `Dialog_DownloadWorkoutsFromSuunto_...`

Reply:
This one is different because it is not part of a dialog. It is the tooltip for the Easy Import Manager entry, as below:

Let me know if you think of a better name and I will change it

![image](https://user-images.githubusercontent.com/25235665/109403338-eb138380-7919-11eb-9926-75aa9a7b3824.png)

Reply:
>This one is different because it is not part of a dialog. 

I've not investigated any detail, I just saw it as a single text key. When this text is related to the easy import launcher page, then the text key should be similar like the other text keys

https://github.com/wolfgang-ch/mytourbook/blob/243549e5e3a21dbb8a0563928add6f479b2bb0ab/bundles/net.tourbook/src/net/tourbook/messages.properties#L1373-L1378

Reply:
Ok, I will use the same name pattern then

Reply:
Fixed in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/310)